tutorials/attribution/intro.md

```diff
@@ -8,10 +8,10 @@ This tutorial walks you through how to set up Snowplow’s Attribution dbt packa
 ### Prerequisites
 
 - [dbt](https://github.com/dbt-labs/dbt) installed (and dbt profile configured for Data Warehouse connection)
-- have the [Campaign attribution enrichment](docs/enriching-your-data/available-enrichments/campaign-attribution-enrichment/) enabled to generate the marketing campaign fields such as mkt_medium, mkt_source, mkt_term, mkt_content, and mkt_campaign which would be needed for channel and campaign classification to ultimately calculate the appropriate attribution
-- have the [Referer parser enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/) enabled to extract attribution data from referer URLs
-- Snowplow Unified Digital [dbt Package]((/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model)) configured and run (v0.4.0 or above) it will generate the following source tables for you:
+- have the [Campaign attribution enrichment](/docs/pipeline/enrichments/available-enrichments/campaign-attribution-enrichment/) enabled to generate the marketing campaign fields such as mkt_medium, mkt_source, mkt_term, mkt_content, and mkt_campaign which would be needed for channel and campaign classification to ultimately calculate the appropriate attribution
+- have the [Referer parser enrichment](/docs/pipeline/enrichments/available-enrichments/referrer-parser-enrichment/) enabled to extract attribution data from referer URLs
+- Snowplow Unified Digital [dbt Package](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model) configured and run (v0.4.0 or above) it will generate the following source tables for you:
   - `snowplow_unified_views` table as a path (touch points) source
   -  `snowplow_unified_conversions` table as a conversions source including the revenue (generated by the optional conversions module)
   - `snowplow_unified_user_mapping` table as the user mapping source (needed for user stitching)
-- (Optional) Marketing Spend Source table. [See more information here](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/#3-channel-spend-information-optional-but-recommended). It will contain your marketing spend data by channel and or campaign with a timestamp field which denotes the period. This is needed for the ROAS calculation within the `snowplow_attribution_overview` model.
+- (Optional) Marketing Spend Source table. [See more information here](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/#3-channel-spend-information-optional-but-recommended). It will contain your marketing spend data by channel and or campaign with a timestamp field which denotes the period. This is needed for the ROAS calculation within the `snowplow_attribution_overview` model.
```

Comment:
```suggestion
- (Optional) Marketing Spend Source table. [See more information here](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md#3-channel-spend-information-optional-but-recommended). It will contain your marketing spend data by channel and or campaign with a timestamp field which denotes the period. This is needed for the ROAS calculation within the `snowplow_attribution_overview` model.
```